tests/test_web_api.py

```diff
@@ -0,0 +1,307 @@
+from hashlib import md5
+from mock import patch
+from os import path as ospath
+from regparser.web.jobs.models import job_status_values
+from regparser.web.jobs.utils import (
+    eregs_site_api_url,
+    file_url,
+    status_url
+)
+from regparser.web.jobs.views import FileUploadView as PatchedFileUploadView
+from rest_framework.test import APITestCase
+from tempfile import NamedTemporaryFile
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
+
+fake_pipeline_id = "f726e1e0-d43c-4eb7-9274-681ddaba427a"
+fake_email_id = "4774f0f6-b53e-4b34-821e-fc8ae5e113fe"
```

Comment:
Would it make sense to use `uuid()` instead? Using specific strings implies that these specific values are meaningful